### DISCUSSION TITLE
Battery for wireless operation

### DISCUSSION THREAD

**Discussion by brassmonk** [FundamentalFrequency/.github#8] [Ideas]
Question for the electrical engineers is there any potential to having an onboard battery for the LMN3 or would the rpi4 draw too much power for this to be viable? I’m considering permanantly affixing a power bank (from just looking at the photos the back of the case seems to have enough space to probably get a thin one in) but I don’t know how long I could potentially use it before it would go flat?

**Comment by HowdyHolto** [comment]
There are several YouTube videos where folks are running Pi4 on battery. They'd need to be fairly beefy and you may have to modify the board to accept a power management kit, but I'm not 100% sure what's already in the works. I'm sure he's thought of this to some extent. That being said... Powerbanks you buy online have management built in of course... Overcharge, status, etc. You could look for a nice way to piggyback one in the meantime.

**Reply by stonepreston** [reply]
The easiest way would be to use a regular power bank (an anker or something) connected directly to the pi. No need to route anything on the PCB

**Reply by HowdyHolto** [reply]
Yeah, I'm thinking this is the best route too... At least early on so we're not overcomplicating early development. There are some super slim packs that pack a decent amount of juice now. I think the only question might be one that provides enough amperage for the Pi to not through low power notifications?
